ports/templates/ports/builds_filtered_table.html

```diff
@@ -54,12 +54,12 @@ <h4 class="text-secondary">Showing {% if port_name != '' %}results for <i>"{{ po
     </thead>
     {% for build in builds %}
         <tr>
-            <td><a href="/ports/{{ build.port_name }}">{{ build.port_name }}</a></td>
+            <td><a href="{% url 'port_detail' build.port_name %}">{{ build.port_name }}</a></td>
             <td>{{ build.builder_name.name }}</td>
```

Comment:
It would make sense to also call a python function here that would turn `builder_name.name` into a shorter version. For anything beyond 10.6 this could always be just OS number.

Reply:
Thank you, I have done it.

Reply:
It is showing some strange behaviour with 10.14, will fix this